Essentials/src/config.yml

```diff
@@ -702,6 +702,9 @@ economy-log-update-enabled: false
 # Minimum acceptable amount to be used in /pay.
 minimum-pay-amount: 0.001
 
+# Enable this to block users who try to /pay another user which ignore them.
+pay-excludes-ignore-list: false
```

Comment:
Wording of this option seems a bit awkward to me (probably just me though).

Reply:
It's a bit confusing to me as well, because I think it can easily be interpreted as doing the opposite of what it does. Maybe something like `allow-pay-by-ignored` would be easier to understand?